src/core/opamStd.ml

```diff
@@ -992,7 +1072,13 @@ module OpamSys = struct
       let cshrc = home ".cshrc" in
       let tcshrc = home ".tcshrc" in
       if Sys.file_exists cshrc then cshrc else tcshrc
+    | SH_pwsh | SH_win_powershell ->
+      if Sys.win32 then win_my_powershell "Microsoft.Powershell_profile.ps1" else
+      List.fold_left Filename.concat (home ".config") ["powershell"; "Microsoft.Powershell_profile.ps1"]
     | SH_sh -> home ".profile"
+    | SH_win_cmd ->
+      (* cmd.exe does not have a concept of profiles *)
+      home ".profile"
```

Comment:
I have some related work on this which I propose coming to also in a subsequent PR - IIUC Diskuv doesn't use `opam init` to do this at all?

Reply:
Is the question whether Diskuv uses `opam init` to populate entries (PATH, auto-complete) in the user's profile? No, Diskuv does not use `opam init` for that.

The installer takes care of putting Opam and with-dkml.exe (the wrapper) and a system dune + utop in the PATH via the registry in the source code https://github.com/diskuv/dkml-component-ocamlcompiler/blob/1e1990447a2fafdca0b2a18e845821952dc0cce2/assets/staging-files/win32/setup-userprofile.ps1#L1538-L1541 .

The uninstaller (on my TODO) will do the reverse.

Reply:
Excellent - the main thing I wanted to be sure was that me messing around with this in a subsequent PR wouldn't risk breaking things for you! I've got a patch which takes advantage of [clink](https://winstall.app/apps/chrisant996.Clink) (if the user has it installed) to do the ".profile" trick for cmd, as it provides start-up scripts in Lua.